app/forms/claims/claim/mentor_training_form.rb

```diff
@@ -78,4 +76,13 @@ def mentor_trainings
   def previous_mentor_training
     mentor_trainings[mentor_trainings.index(mentor_training) - 1]
   end
+
+  def training_allowance
+    @training_allowance ||= Claims::TrainingAllowance.new(
+      mentor:,
+      provider:,
+      academic_year: Claims::ClaimWindow.current.academic_year,
```

Comment:
Technically speaking, this _could_ be incorrect, right?

Excluding the external constraints, being unable to edit/submit a claim when the window is closed, it would be better to contain the responsibilities of those modules, right?

Reply:
@Nitemaeric resolved by 0b2ea03cb41652c5a67a508e548c2cf4a7b41c07 🙂